app/src/app/job/hybrid.rs

```
@@ -589,6 +589,7 @@ mod tests {
     fn create_test_app(
         use_mock_id: bool,
     ) -> Result<(HybridJobAppImpl, RedisJobResultPubSubRepositoryImpl)> {
+        std::env::set_var("PLUGINS_RUNNER_DIR", "../target/debug");
```

maybe fail in not set environment (ex. without .env)